tests/001-test-use-devs-to-create-thin-pool.sh

```diff
@@ -17,6 +17,7 @@ test_devs() {
   cat << EOF > /etc/sysconfig/docker-storage-setup
 DEVS="$devs"
 VG=$vg_name
+CONTAINER_THINPOOL=docker-pool
```

Comment:
I had objected to third mode that is if input file name is "/etc/sysconfig/docker-storage-setup" then pool name will be docker-pool.